<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(152, 228, 255);</string>
  </property>
  <widget class="QPushButton" name="bt_codigoMat">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>imagenes/baseline-search.png</normaloff>imagenes/baseline-search.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_codigoSalon">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>280</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>imagenes/baseline-search.png</normaloff>imagenes/baseline-search.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_CedulaProf">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>340</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>imagenes/baseline-search.png</normaloff>imagenes/baseline-search.png</iconset>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="boton_guardar">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:15px;
color:white;
font-size:15px;
background-color: rgb(104, 126, 255);</string>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="boton_cancelar">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:15px;
color:white;
font-size:15px;
background-color: rgb(104, 126, 255);</string>
      </property>
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>261</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Dia :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="input_dia"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Hora :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="input_hora"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Codigo Materia</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="input_materia"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Codigo Salon</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="input_salon"/>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Nombre Profesor</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="input_profesor"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
